### PR TITLE
Bump Kyma to version 1.8.0

### PR DESCRIPTION
**Pull request by hudymi**
**Description**

Changes proposed in this pull request:

- Bumped

**Related issue(s)**

### CODE REVIEW

File: google-cloud/schema.yaml [hunk 1]
```diff
@@ -18,8 +18,7 @@ properties:
     title: Cluster Admin Service Account
     description: >-
       Name of a service account in the target namespace that
```
Comment (majakurcius) [inline google-cloud/schema.yaml]:
```suggestion
      Name of a service account in the target Namespace that
```

File: google-cloud/schema.yaml [hunk 2]
```diff
@@ -18,8 +18,7 @@ properties:
...
-      has cluster-admin permissions. This is needed for the operator
-      to create Spark CRD and jobs.
+      has cluster-admin permissions. This is needed for the installation of Kyma.
```
Comment (majakurcius) [inline google-cloud/schema.yaml]:
```suggestion
      has cluster-admin permissions. Required to install Kyma.
```

**Review by majakurcius — APPROVED** [review]
Just a minor detail. Otherwise lgtm:)

**Review by magicmatatjahu — APPROVED** [review]
LGTM

**Pull request merged**